website/docs/d/identity_group.html.markdown

```diff
@@ -0,0 +1,47 @@
+---
+layout: "azuredevops"
+page_title: "AzureDevops: azuredevops_identity_group"
+description: |-
+  Use this data source to access information about existing Groups within Azure DevOps
+---
+
+# Data Source: azuredevops_identity_group
+
+Use this data source to access information about existing Groups within Azure DevOps On-Premise(Azure DevOps Server).
+
+## Example Usage
+
+```hcl
+data "azuredevops_project" "example" {
+  name = "Example Project"
+}
+
+# load all existing groups inside an organization
+data "azuredevops_identity_group" "example-all-group" {
+  name = "Group-Name"
+}
+
+# load all existing groups inside a specific project
+data "azuredevops_identity_group" "example-project-group" {
+  project_id = data.azuredevops_project.example.id
+  name = "[Project-Name]\\Group-Name"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+- `name` - (Required) The name of the group.
+- `project_id` - (Optional) The Project ID. If no project ID is specified all groups of an organization will be returned
```

Comment:
wrong description, no `project_id`  this resource search the group at organization level?